### ISSUE TITLE
Custom settings are beeing ignored

### ISSUE THREAD

**Issue by Geddo** [apache/cordova-electron#103]
My setup looks like this:
**config.xml**

    <platform name="electron"> 
        <preference name="ElectronSettingsFilePath" value="www/platforms/electron.json" />
        <icon src="www/assets/img/icon.png" />
    </platform>
**www/platforms/electron.json**

    "browserWindow": {
            "width": 1280,
            "height": 720,
            "resizable": false,
            "fullscreen": true,
            "webPreferences": {
              "devTools" : false,
              "plugins" : true          
            }
        },  
        "build": {
            "asar": false,
            "electronVersion": "2.0.18",
            "electronDownload": {
              "version": "2.0.18"
            }
        }
Sadly "devTools" : false is getting ignored, same for the electron-Version.  As I can read in the docs, I need --release to have no devTools, but how can I change the used electronVersion?
When I run "cordova build electron" I'm getting this summary:

    • electron-builder version=20.44.4
    • writing effective config file=platforms\electron\build\builder-effective-config.yaml
    • installing production dependencies platform=win32 arch=x64 appDir=...
    • packaging platform=win32 arch=x64 electron=4.0.1 appOutDir=...
But where can I change the options for the electron(version), appDir etc?
Thank you for your help

**Comment by GedasGa** [comment]
Hi @Geddo, sorry for a late reply. 

`devTools` options in the "Electron Settings File" **<** flag in CLI

So if they contradict one each other, flag in CLI takes the upper hand. And on Debug Builds, DevTools are shown by default (without a flag or with `--debug`).

Regarding to the second part of your question, currently, it's not possible to change electron version in the "Electron Settings File". If you really want to do that, please update the electron dependency version in the `package.json` and `bin/templates/cordova/lib/build/base.json` files.

> Note: We never tested the compatibility with other electron releases other than 4.0.1.

**Comment by erisu** [comment]
As described above this flag can not be set through configs and is controlled by the CLI.

E.g.

```shell
cordova build electron
cordova build electron --debug
```

Both commands above will set the `devTools` flag to `true` because.

```shell
cordova build electron --release
```

The above command will set the `devTools` flag to `false`

And as @GedasGa pointed out regarding the second part of your question, there is no way to change the Electron version. There is an open ticket where someone suggested this feature, but as of now, it is not available.

And, as a side note, Cordova Electron 3.0.0 is released which supports Electron 14.0.0.
